quupe-RN/js/navigation/NavigationLayout.js

```diff
@@ -60,6 +62,20 @@ const accountStack = createStackNavigator(
   }
 );
 
+const signInStack = createStackNavigator({
+  SignIn: SignInContainer
+})
+
+const authStack = createSwitchNavigator(
```

Comment:
Nice work! let's actually call this authSwitch. const authSwitch = createSwitchNavigator( ...